plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt

```diff
@@ -63,6 +63,6 @@ open class KtlintBasePlugin : Plugin<Project> {
     }
 
     companion object {
-        const val LOWEST_SUPPORTED_GRADLE_VERSION = "6.0"
+        const val LOWEST_SUPPORTED_GRADLE_VERSION = "6.8"
```

Comment:
Is this consistent with the Android Gradle Plugin?

Reply:
Well, I am not sure what you mean, but we need at least 6.8 version of gradle to make it work with ktlint `0.43.2`.
Here is the table
<img width="902" alt="image" src="https://user-images.githubusercontent.com/15987499/182235454-6ca45243-9b31-49ea-95cc-f9443738d0ad.png">